pennylane/operation.py

```diff
@@ -1350,20 +1501,25 @@ def diagonalizing_gates(self):
 
         return diag_gates
 
-    @property
-    def matrix(self):
-        r"""Matrix representation of the tensor operator
+    def matrix(self, wire_order=None):
+        r"""Matrix representation of the Tensor operator
```

Comment:
I can't quite tell from the github view, but what class is this in/how come it is here as well as above?

Reply:
This is the `Tensor` class, where a lot of different stuff happens :) Hopefully deleted soon!

Above method is in another class!

Reply:
got it!